src/game.c

```diff
@@ -946,6 +950,14 @@ void game()
                 gamepause = 0;
             }
 
+            int speed_adjust_delta = (key(SDLK_F9) ? -5 : key(SDLK_F10) ? +5 : 0);
```

Comment:
I think this code should be in wport's `handle_special_key` function. Then speed_adjust_delta won't be needed either.